tools/setup-windows/setup-windows.csproj

```diff
@@ -1,38 +1,20 @@
 <?xml version="1.0" encoding="utf-8"?>
-<Project DefaultTargets="Build" ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+<Project>
+  <!--NOTE: manually importing Sdk.props and Sdk.targets so we can replace the Pack target-->
+  <Import Sdk="Microsoft.NET.Sdk" Project="Sdk.props" />
```

Comment:
You should just do this for this project:
```xml
<Project Sdk="Microsoft.NET.Sdk">
```
And remove any `<Import Sdk="Microsoft.NET.Sdk" />`  in here.

I only had to do this in AndroidX or other repos, because redth had a `Pack` target like this:
```xml
<Target Name="Pack" />
```
And he needed to actually make the `Pack` target blank. To do that you have to `<Import/>` first then replace the target second.

Reply:
I thought this was some hack for whatever reason.

Reply:
Added 
```
<Project Sdk="Microsoft.NET.Sdk">
```